src/components/LogView/LogLineText.js

```diff
@@ -56,11 +56,7 @@ export function findJSONObjectsInLine(text: string): string[] {
 
 export default class LogLineText extends React.PureComponent<Props> {
   lineRef: ?HTMLSpanElement = null;
-  prettyPrintedText: string[] = [];
-
-  componentDidMount() {
-    this.prettyPrintedText = findJSONObjectsInLine(this.props.text);
-  }
+  prettyPrintedText: string[] = findJSONObjectsInLine(this.props.text);
```

Comment:
performing this action here as opposed to in `componentDidMount` is going to have a different outcome because they are performed at different times in the component lifecycle. If you move `findJSONObjectsInLine(this.props.text);` back into `componentDidMount` does it still cause issues?

Reply:
the component lifecycle thing makes sense. and yes it still causes issues- that was the original location of the function call that made the pretty printing to collapse on scrolling.

Reply:
was this an existing bug or is this a bug that was introduced while you were developing?

if it's the latter then what i would do is revert the code as much as you can back to a version before this bug existed / when this feature worked as it's supposed to. from there you can incrementally introduce the code changes you made until something breaks - and then you'll know what caused the bug. 

if you feel out of options or defeated then ping me and I am happy to pair program on zoom to figure out the issue.

Reply:
I'm very confident this is something I introduced since I created the pretty print option. I know what component is causing the problem, since it's the only one that had major updates, but the root source of the issue is not at all evident to me. I'll go through and backtrack as much as I can, then let you know if I'm still stumped.